mobile_verifier/src/boosting_oracles/data_sets.rs

```diff
@@ -507,6 +502,73 @@ impl DataSetType {
     }
 }
 
+#[async_trait::async_trait]
+pub trait DataSetProcessor: Send + Sync + 'static {
+    async fn set_all_oracle_boosting_assignments(
+        &self,
+        pool: &PgPool,
+        data_sets: &HexBoostData<impl HexAssignment, impl HexAssignment, impl HexAssignment>,
+    ) -> anyhow::Result<()>;
+
+    async fn set_unassigned_oracle_boosting_assignments(
+        &self,
+        pool: &PgPool,
+        data_sets: &HexBoostData<impl HexAssignment, impl HexAssignment, impl HexAssignment>,
+    ) -> anyhow::Result<()>;
+}
+
+#[async_trait::async_trait]
+impl DataSetProcessor for FileSinkClient {
+    async fn set_all_oracle_boosting_assignments(
```

Comment:
The functions this trait is replacing are unused, can they be removed?

Reply:
oop yup, my bad